iXBRLViewerPlugin/viewer/src/js/fact.js

```diff
@@ -100,6 +100,13 @@ Fact.prototype.readableValue = function() {
         /* Replace runs of whitespace (including nbsp) with a single space */
         v = html.text().replace(/[\u00a0\s]+/g, " ").trim();
     }
+    else if (this.isEnumeration()) {
+        var labels = [];
+        for (const qn of v.split(' ')) {
+            labels.push(this._report.getLabel(qn, 'std') || qn);
```

Comment:
Should we wait on the other PR and `getLabelOrName` here?

Reply:
Yes, good point.  I've now merged the other branch into this one and made that change, so this PR is currently showing the changes in both, but it should go back to just this once #224 is merged.